src/content/getting-started/cloud-provider-accounts/vmware-cloud-director/_index.md

```diff
@@ -0,0 +1,62 @@
+---
+linkTitle: Preparing VMware Cloud Director for CAPI
+title: Preparing VMware Cloud Director for Cluster API
+description: How to set up your VMware Cloud Director Cluster tenant to run Giant Swarm management clusters and workload clusters under your jurisdiction.
+weight: 80
+user_questions:
+  - What are the VMware Cloud Director prerequisites for use with Giant Swarm?
+  - What do I need to configure in VMware Cloud Director in order to run Giant Swarm clusters?
+  - What user permissions do I need to run Giant Swarm clusters in VMware Cloud Director?
+menu:
+  main:
+    identifier: gettingstarted-infraprovider-clouddirector
+    parent: gettingstarted-infraprovider
+last_review_date: 2023-10-03
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-rocket
+---
+In order to run the Giant Swarm platform in your VMware Cloud Director (VCD) environment, a number of prerequisites must be satisfied to support Cluster API Provider VMware Cloud Director (CAPVCD).
```

Comment:
```suggestion
In order to run the Giant Swarm platform in your VMware Cloud Director (VCD) environment, a number of prerequisites must be satisfied to support creating workload clusters with Cluster API Provider VMware Cloud Director (CAPVCD).
```

Reply:
It's prerequisites for CAPVCD so they apply to being able to run the provider in general.